src/modules/TeamsList/components/Teams/components/MyTeam/MyTeam.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { FC, useState } from 'react';
+import { StyledMyTeam, HeaderDecor, TableWrapper } from './styled';
+import { Button } from 'typography';
+import { DARK_TEXT_COLOR, WHITE_COLOR } from 'appConstants/colors';
+import { Team } from 'types';
+import { MembersListToggle } from '../MemberListToggle/MemberListToggle';
```

Comment:
Name file index.tsx and change import to import { MembersListToggle } from '../MemberListToggle';

Reply:
done